schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -1005,7 +1005,8 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2018-06-01/Microsoft.Security.json#/resourceDefinitions/informationProtectionPolicies" },
                   { "$ref": "https://schema.management.azure.com/schemas/2019-03-01/Microsoft.Insights.json#/resourceDefinitions/actionGroups" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-06-01-preview/Microsoft.ManagedServices.json#/resourceDefinitions/registrationAssignments" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2018-06-01-preview/Microsoft.ManagedServices.json#/resourceDefinitions/registrationDefinitions" }
+                  { "$ref": "https://schema.management.azure.com/schemas/2018-06-01-preview/Microsoft.ManagedServices.json#/resourceDefinitions/registrationDefinitions" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2018-09-01-preview/Microsoft.BareMetal.json#/resourceDefinitions/crayServers" }
```

Comment:
>crayServers [](start = 138, length = 11)

we have a new root schema: 2019-04-01/deploymentTemplate.json. can you update that schema as well?

Reply:
updated